lambda_backend/route_microservice/src/addComment.ts

```diff
@@ -27,10 +26,20 @@ const addComment: Handler = async (event: AddCommentEvent) => {
   const accessToken = Authorization.split(' ')[1];
   const { username } = (await jwt_decode(accessToken)) as JwtPayload;
   let { comments } = Item;
+
+  const isCommentLimitReached =
+    comments.filter((comment) => username === comment.username).length >= COMMENT_LIMIT;
```

Comment:
Usually limit should be inclusive? `>` instead of `>=`?

Reply:
If current commentCounts is 1 and comment limit set is 1, stop the function

Reply:
so if number of comments is 1 then `isCommentLimitReached = true` ?